security/middleware.py

```diff
@@ -214,6 +219,14 @@ class XssProtectMiddleware(BaseMiddleware):
 
     DEFAULT = 'sanitize'
 
+    def __init__(self, get_response=None):
+        super().__init__(get_response)
+        warnings.warn('The middleware "{name}" will no longer be supported in '
+        'future releases of this library. Refer to {url} for an '
+        'alternative approach.'.format(
```

Comment:
Could we provide a little more detail such as mentioning the django settings that replace the middleware? Can keep the link to the docs as an additional info link.